### PR TITLE
Revert "Disable Horizon service temporarily"

### PR DESCRIPTION
**Pull request by mcgonago**
This reverts commit 95e301f58fc066c3dc40a01426985f66dede49f6.

### CODE REVIEW

**Review by bshephar — APPROVED** [review]

**Comment by abays** [comment]
Has anyone tested this revert in a `uni04delta-ipv6` downstream job?

File: dt/uni04delta-ipv6/kustomization.yaml [hunk 1]
```diff
@@ -194,15 +194,3 @@ replacements:
           - spec.nova.template.cellTemplates
         options:
           create: true
-
-  - source:
```
Comment (fultonj) [inline dt/uni04delta-ipv6/kustomization.yaml:198]: we should keep this

File: examples/dt/uni04delta-ipv6/control-plane/service-values.yaml [hunk 2]
```diff
@@ -8,9 +8,6 @@ metadata:
 data:
   preserveJobs: false
 
-  horizon:
-    enabled: false
```
Comment (fultonj) [inline examples/dt/uni04delta-ipv6/control-plane/service-values.yaml:12]: we just need to change this to true

**Comment by fultonj** [comment]
Replaced by https://github.com/openstack-k8s-operators/architecture/pull/342

File: dt/uni04delta-ipv6/kustomization.yaml (hunk 1 above)
Comment (sdatko) [reply dt/uni04delta-ipv6/kustomization.yaml:198]: Imo we should not – in other uni jobs we do not need to specify that explicitly, it gets the default from lib/control-plane/openstackcontrolplane.yaml

File: examples/dt/uni04delta-ipv6/control-plane/service-values.yaml (hunk 2 above)
Comment (sdatko) [reply examples/dt/uni04delta-ipv6/control-plane/service-values.yaml:12]: (see https://github.com/openstack-k8s-operators/architecture/pull/341/files#r1687774153)

**Comment by sdatko** [comment]
IMVHO This one is preferred over https://github.com/openstack-k8s-operators/architecture/pull/342

In yesterday build we tested that the Horizon deploys fine in IPv6 environment now.

**Review by fultonj — APPROVED** [review]
/lgtm
/approve

**Pull request merged**